### ISSUE TITLE
Widebody version

### ISSUE THREAD

**Issue by Ashram56** [jonaskello/wpc-cabinet#2]
Good afternoon,

Would you have by chance a Widebody IJ model ? I'd like to renovate my IJ, and since I have access to a CNC router I'd very much like to build the cabinet from scratch if possible

Cheers

**Comment by jonaskello** [comment]
Hi there! 

Yes I actually have an IJ widebody, and also a STTNG widebody :-). The measurements in this plan comes from them but I've adjusted the width to standard body. I have not got around to building a cabinet from these plan yet so I cannot vouch for the accuracy though I did spend a lot of time trying to make them as accurate as possible.

**Comment by Ashram56** [comment]
Thanks,

Did you create a model using the widebody measurements ? Since this model is standard size, I can't really use it for my Indy...

Also, do you have the mounting points locations for Indy and STTNG (for the playfield hinges) ?

Regards

Cheers

**Comment by Ashram56** [comment]
Good morning, coming back on this topic, did you create a model using the original widebody measurements ? And any indication on the playfield hinges location ?

Cheers

**Comment by jonaskello** [comment]
Yes, the measurements are from my original widebody games. However I then changed the width so the plans are not widebody in width but instead standard WPC width.

I'm not 100% certain but I think the playfield hinges can be at different locations for different games. I only have one widebody at home currently, the other one is at work so I cannot compare the hinges positions.

**Comment by jonaskello** [comment]
Actually I saw now that I noted everything in the readme so you can just read this section :-)

https://github.com/jonaskello/wpc-cabinet#notes

**Comment by jonaskello** [comment]
If you are looking for widebody measurements you can take the plans and just change the width back to widebody. Then you will have 100% original widebody measurements as all measurements are from widebody games and only the width was changed.

**Comment by Ashram56** [comment]
With regards to the hinges, yes they are at different location.  That's the main problem, because it's rather hard to take a measurement of the hinge holes without disassembling first the original cabinet (specifically interested into Indy).

With regards to dimension, if only the width has been changed then indeed it's rather straightforward. I was thinking there might be other dimensions
